docs/ta/docs/index.md

```diff
@@ -0,0 +1,463 @@
+
+{!../../../docs/missing-translation.md!}
+
+
+<p align="center">
+  <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
+</p>
+<p align="center">
+    <em>FastAPI வரைச்சட்டம், அதிக வீரியம், கற்பதற்க்கு எளியது, வேகமாக குறிமுறையாக்கல், உடன் பயனுக்கு தயாரானது</em>
+</p>
+<p align="center">
+<a href="https://github.com/tiangolo/fastapi/actions?query=workflow%3ATest+event%3Apush+branch%3Amaster" target="_blank">
+    <img src="https://github.com/tiangolo/fastapi/workflows/Test/badge.svg?event=push&branch=master" alt="Test">
+</a>
+<a href="https://codecov.io/gh/tiangolo/fastapi" target="_blank">
+    <img src="https://img.shields.io/codecov/c/github/tiangolo/fastapi?color=%2334D058" alt="Coverage">
+</a>
+<a href="https://pypi.org/project/fastapi" target="_blank">
+    <img src="https://img.shields.io/pypi/v/fastapi?color=%2334D058&label=pypi%20package" alt="Package version">
+</a>
+<a href="https://pypi.org/project/fastapi" target="_blank">
+    <img src="https://img.shields.io/pypi/pyversions/fastapi.svg?color=%2334D058" alt="Supported Python versions">
+</a>
+</p>
+
+---
+
+**ஆவணம் (docuemntation - doubtful)**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+
+**முலக்குறியீடு**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
+---
+
+FastAPI ஒரு நவீன, வேகமான (அதிக செயல் திறன்), வலைத்தள வரைச்சட்டம். இது பைத்தான் 3.7+ல் APIக்களை பைத்தான் வகை குறிப்புகள் (type hints) மூலமாக உருவாக்க உதவுகிறது.
+
+இதன் முக்கய அம்சங்கள்:
+
+* **வேகம்**: அதிக செயல் திறன், **NodeJS** மற்றும் **Go**விற்க்கு ஒப்பிடாக (ஸ்டார்லெட் மற்றும் பைடான்டிக்கு நன்றிகள்). [பைத்தான் கட்டமைப்புகளிள் மிக வேகமான ஒன்று](#செயல்-திறன்)
+* **வேகமாக குறியீடாக்குதல்**: பண்புகூறுகள் (features) உருவாக்கும் வேகத்தை 200% முதல் 300% அதிகரிக்கிறது. *
+* **குறைந்த பிழை தவறுகள்**: ஏறக்குறைய 40% உருவாக்குனர்களால் வரும் பிழைகளை குறைக்கிறது.
+* **உள்ளுணர்வு**: சிறந்த தொகுப்பி துணை. எங்கும்<abbr title="தானே முடித்தல்"> முடித்தல் </abbr>. தவறு நீக்குதலில் (debugging) குறைந்த நேரம் செலவீட்டல்.
+* **எளியது**: கற்பதற்க்கம் பயன்பாட்டிற்க்கும் எளிமையாக வடிவமைக்கபட்டது.  ஆவணப்படித்தலில் குறைந்த நேரம்.
+* **குட்டி**: கம்மியான குறீயிடு இரட்டிப்பு. ஒவ்வொரு அளவுருவின் (parameter) அறிவிப்பிலும் பண்முக பண்புகூறுகள். குறைந்த பிழை தவறுகள்
+* **வீரியம்**: உற்பத்தி ஒட்டதிற்கான குறியீடுகளை பெறுங்கள். கூடுதலாக தானியக்க ஊடாடும் ஆவணம்.
+* **நிலையானது**: திறந்த API அடிப்படையான <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (முன்னர் Swagger என்று அழைக்கப்பட்டது) மற்றும் <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>வை சார்ந்த்து (மற்றும் முழுமையாக ஒற்றுமைக்கொண்டது.)
```

Comment:
This could be Open API instead of திறந்த API.
அடிப்படையான -> அடிப்படையிலான

Open API அடிப்படையிலான <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (முன்னர் Swagger என்று அழைக்கப்பட்டது) மற்றும் <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>வை சார்ந்தது (மற்றும் முழுமையாக ஒற்றுமைக்கொண்டது.)